code/obj/machinery/computer/syndicate/podbay.dm

```diff
@@ -0,0 +1,104 @@
+// Riot computer but reflavored for the Cairngorm
+
+/obj/machinery/computer/battlecruiser_podbay
+	name = "podbay authorization"
+	icon_state = "drawbr"
+	desc = "Controls access to the podbay. Use this when your team is ready to go."
+	density = FALSE
+	/// How many authentications are needed to release the shielding on the podbay. Set the first time someone clicks this.
+	var/auth_need
+	var/list/authorized
+
+	light_r = 1
+	light_g = 0.3
+	light_b = 0.3
+
+	/// Whether or not the podbay on the Cairngorm is authorized.
+	var/authed = FALSE
+
+	initialize()
+		SPAWN(10 MINUTES)
+			authorize() // If they haven't done it before the ten minute mark-ish, do it for em
+		..()
+
+	proc/authorize()
+		if (src.authed)
+			return
+		logTheThing(LOG_STATION, usr, "authorized Cairngorm podbay access")
+		src.authed = TRUE
+		src.ClearSpecificOverlays("screen_image")
+		src.icon_state = "drawbr-alert"
+		src.UpdateIcon()
+
+		for_by_tcl(forcefield, /obj/forcefield/battlecruiser)
+			qdel(forcefield)
+
+		SPAWN(0.5 SECONDS)
+			var/operative_mobs = list()
+			for (var/datum/antagonist/operative as anything in (get_all_antagonists(ROLE_NUKEOP) + get_all_antagonists(ROLE_NUKEOP_COMMANDER)))
+				operative_mobs += operative.owner.current
+			boutput(operative_mobs,"<b>The podbay has been authorized. You may now leave the Cairngorm using your pods!</b>",forceScroll=TRUE)
+			playsound_global(operative_mobs, 'sound/vox/pods.ogg', 50, vary=FALSE)
+			sleep(1 SECONDS)
+			playsound_global(operative_mobs, 'sound/vox/authorized.ogg', 50, vary=FALSE)
```

Comment:
Regarding the VOX, this makes the pod bay opening sound like a global announcement, as VOX is typically reserved for such things. I think the `boutput()` suffices for such an announcement.

Reply:
There are other items in the game that use VOX, such as the HoS's lawbringer, which are not global. I feel VOX is important to keep, for two reasons:
1. Chat can easily get filled up by someone in the firing range, or be ignored altogether
2. It is important that people on the team know when they're ready to move out.
I suppose one unrealistic part of this is that operatives outside of the cairngorm will hear this, but I don't mind that personally.

Reply:
I've removed it but I would like to keep it

Reply:
Feel free to readd it; my comment above voices my own thoughts on the matter of the VOX, and is just a suggestion. Ultimately it is the developers who have final say over what should and what should not be part of the game.

Reply:
Gocha, I'll re-add it and see what comes of it.